coalib/bears/Bear.py

```diff
@@ -16,14 +16,31 @@
 from coalib.bears.BEAR_KIND import BEAR_KIND
 from coalib.output.printers.LogPrinter import LogPrinterMixin
 from coalib.results.Result import Result
+from coalib.misc.Constants import FALSE_STRINGS, TRUE_STRINGS
 from coalib.results.TextPosition import ZeroOffsetError
 from coalib.settings.FunctionMetadata import FunctionMetadata
 from coalib.settings.Section import Section
 from coalib.settings.ConfigurationGathering import get_config_directory
+from coalib.settings.Setting import typed_list
 
 from .meta import bearclass
 
 
+def _debugger_helper(bear):
+    if bear is None:
+        return False
+    elif 'debug_bears' not in bear.section or (
+            bear.section['debug_bears'].value.lower() in FALSE_STRINGS):
+        return False
+    elif bear.name.lower() in (
+            map(str.lower, typed_list(str)(bear.section['debug_bears']))) or (
+                typed_list(str)(bear.section['debug_bears'])[0].lower() in
+            TRUE_STRINGS):
+        return True
+    else:
+        return False
```

Comment:
you can return the result of `bear.name.lower() in ...` directly instead of using an if, it's like `if True: return True else: return False` now

Reply:
@Makman2 , so we need to enhance coala_util's `FALSE_STRINGS` ?  better create an issue quick as those libraries need to get released soon

Reply:
Can you review this code :sweat_smile: 
I will create a issue after getting green from Makman

Reply:
If possible I will solve that :+1: